src/gt4py/next/program_processors/runners/dace_fieldview/transformations/strides.py

```diff
@@ -224,8 +223,13 @@ def gt_map_strides_to_dst_nested_sdfg(
 ) -> None:
     """Propagates the strides of `outer_node` along `edge` along the dataflow.
 
-    For more information see the description of `_gt_map_strides_to_nested_sdfg_src_dst().
-    However it is recommended to use `gt_propagate_strides_of()` directly.
+    In this context "along the dataflow" means that `edge` is an outgoing
+    edge of `outer_node` and the strides are into all NestedSDFGs that
+    are downstream of `outer_node`.
+
+    Except in certain cases this function should not be used directly. It is
+    instead recommended to use `gt_propagate_strides_of()`, which propagates
```

Comment:
It is not the access node version, because that thing does not handle all AN in the SDFG.
For a (stupid, aka. myself) I want that `gt_propagate_strides_of()` is used that acts on the whole SDFG.